### PR TITLE
Added base climber scaffold

### PR DESCRIPTION
**Pull request by ghost**
Added scaffold for the climber component

### CODE REVIEW

File: components/climb.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from magicbot import tunable
+import wpilib
+
+
+class Climber:
+    shooter_motor: wpilib.Talon
```
Comment (outsidermm) [inline components/climb.py:6]: can we preassume the hardware for the climber

File: components/climb.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    shoot_speed = tunable(1.0)  # speed is tunable via NetworkTables
```
Comment (outsidermm) [inline components/climb.py:7]: Are you sure we are even going to have a motor-powered climber mech?

File: components/climb.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+    def __init__(self):
+        self.deployed = False
+
+    def deploy(self):
+        """Causes the climber to do it's thing"""
+        self.deployed = True
+
+    def is_ready(self):
```
Comment (outsidermm) [inline components/climb.py:16]: I think this is a feedback function, magicbot feedback decorator?

**Review by outsidermm — COMMENTED** [review]
syntaxical annotation please
especially the return type for methods

**Review by james-ward — APPROVED** [review]
LGTM

**Pull request merged**